org.eclipse.xtend.core/build.properties

```diff
@@ -13,6 +13,5 @@ exclude.. = **/GenerateXtend.mwe2
 src.includes = about.html
 src.excludes = generator/,\
                generator-xtend-gen/
-additional.bundles = org.eclipse.emf.mwe2.launch,\
```

Comment:
did you try if run workflow works? for me it does not.
![grafik](https://github.com/eclipse/xtext/assets/238736/4d1ddb24-d50c-4653-abde-d6d580803837)

Reply:
Of course, my bad
We need to port the mwe2 file (back) to Java

Reply:
or we keep launch here

Reply:
Yes, I just pushed that. I misread the error message. Multitasking is hard.